tools/bin/ci_check_dependency.py

```diff
@@ -1,253 +0,0 @@
-import sys
-import os
-import os.path
-import re
-from typing import Any, Dict, Text, List
-import requests
-
-CONNECTOR_REGISTRY_URL = "https://connectors.airbyte.com/files/registries/v0/oss_registry.json"
-CONNECTORS_PATH = "./airbyte-integrations/connectors/"
-NORMALIZATION_PATH = "./airbyte-integrations/bases/base-normalization/"
-DOC_PATH = "docs/integrations/"
-IGNORE_LIST = [
-    # Java
-    "/src/test/","/src/test-integration/", "/src/testFixtures/",
-    # Python
-    "/integration_tests/", "/unit_tests/",
-    # Common
-    "acceptance-test-config.yml", "acceptance-test-docker.sh", ".md", ".dockerignore", ".gitignore", "requirements.txt"]
-IGNORED_SOURCES = [
-    re.compile("^source-e2e-test-cloud$"),
-    re.compile("^source-mongodb$"),
-    re.compile("^source-python-http-tutorial$"),
-    re.compile("^source-relational-db$"),
-    re.compile("^source-stock-ticker-api-tutorial$"),
-    re.compile("source-jdbc$"),
-    re.compile("^source-scaffold-.*$"),
-    re.compile(".*-secure$"),
-]
-IGNORED_DESTINATIONS = [
-    re.compile(".*-strict-encrypt$"),
-    re.compile("^destination-dev-null$"),
-    re.compile("^destination-scaffold-destination-python$"),
-    re.compile("^bases-destination-jdbc$")
-]
-COMMENT_TEMPLATE_PATH = ".github/comment_templates/connector_dependency_template.md"
-
-
-def download_and_parse_registry_json():
-    response = requests.get(CONNECTOR_REGISTRY_URL)
-    response.raise_for_status()
-    return response.json()
-
-
-def main():
```

Comment:
@alafanechere Just want to double check my reading here. Affected connectors seems to only look at dependant connectors through gradle (i.e. only java)

So there never was logic to find related python connectors when a base or similar changed.

so we can remove safely

Reply:
I would explain why we only see this message on java prs :D